app/views/tag/_tags.html.erb

```diff
@@ -1,34 +0,0 @@
-<ul class= "list-inline">
```

Comment:
Why have you deleted this file?
I think you need to untrack this file instead of deleting it. Any problem?

Reply:
Oh I am sorry, could you please instruct me on how to restore it and untrack it after restoration?

Reply:
https://stackoverflow.com/questions/38743912/git-remove-a-file-from-a-pull-request will help you